packages/bridge/src/utils/assets.ts

```diff
@@ -74,3 +74,6 @@ export const filterModalEthTokens = (tokens: TokenInfo[]) => {
     token => EXCLUDED_ETH_TOKENS.indexOf(token.symbol.toLowerCase()) < 0,
   );
 };
+
+export const SPLtokenAddress = 'E5ndSkaB17Dm7CsD22dvcjfrYSDLCxFcMd6z8ddCk5wp';
```

Comment:
Let's please use more descriptive name for the token, e.g. RIN_SOLANA_MINT and WWT_RIN_ETHEREUM_MINT.